Controllers/ServiceController.cs

```diff
@@ -1,16 +1,205 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Configuration;
+using System.Data.SqlClient;
 using System.Linq;
 using System.Net;
 using System.Net.Http;
 using System.Web.Http;
+using ulacit_bnb.Models;
 
 namespace ulacit_bnb.Controllers
 {
     [Authorize]
     [RoutePrefix("api/service")]
     public class ServiceController : ApiController
     {
+        //GET GetId
```

Comment:
Are this comments really necessary? we already have the HTTPAction above.
If the purpose is to make clear the division between the methods maybe we could use something like this:
```c#
// ---------------------------------------------------
```